src/main/java/org/opendatakit/suitcase/ui/PullPanel.java

```diff
@@ -138,17 +159,42 @@ public void actionPerformed(ActionEvent e) {
         pullButtonPanel.add(sPullButton);
     }
 
-    public void setButtonsState(ButtonState pullButtonState,ButtonState refreshButtonState) {
+    public void setButtonsState(ButtonState pullButtonState, ButtonState refreshButtonState) {
         sPullButton.setEnabled(pullButtonState.getButtonStateBooleanValue());
         sRefreshButton.setEnabled(refreshButtonState.getButtonStateBooleanValue());
     }
 
+    private void logout(){
+        File propFile = new File(SuitcaseConst.PROPERTIES_FILE);
+        if (propFile.exists()) {
+            Properties appProperties = new Properties();
+            try (FileInputStream fileInputStream = new FileInputStream(SuitcaseConst.PROPERTIES_FILE)) {
+                appProperties.load(fileInputStream);
+                appProperties.clear();
```

Comment:
Why deleting stored data after logout? If the user logouts and closes the application after that when the user tries to log in again we still want its data to be there. Right?

Reply:
Yes sir I am modifying it now. Only password will be removed.

Reply:
Yes. So I think correct behavior should be like this:
If the save password box is unchecked - remove the password after logout
If the save password box is checked - No need to remove anything

Reply:
Sir the password is not saved in the properties file after login if save password is unchecked. So one way is: 
 1. checking that password field is there or not  
 2.  Remove if it is there.
 If any other implementation would be more efficient please suggest.

Reply:
So in that case, you don't need to change the properties file at all after the user logout. 
If the user checked the save password box password will be stored in the properties file and we can reuse it when the user restarts the application.
In case of save password box is unchecked, password will not be stored in the properties file which is what we want.

Reply:
Yes sir but this will work but only if user stays in the application after logout. Because after user opens the application again auto login logic is implemented in this way:
1. If password field is present login the user
2. Else if password field is absent only fill the server_url, app_id and username and leave password text field blank.

Reply:
So after we remove password it will not login the user when application is opened again.

Reply:
But we want a password to be filled in automatically when the user opens the application again and he previously had checked the save password box.  Right?
A simple solution which I can think of,

Start of application: If the properties file is present fill all the fields using values in the properties file, else create properties file with all the keys, and values set to an empty string. Run Auto login logic only when password is not an empty string. 
After Successful login: Change all the values in the properties file corresponding to user-provided values. If the save password box is not checked set the value of the password(in the properties file) to an empty string.
After Logout: No need to change anything in the properties file.

It's just a suggestion. If you think there is some easier way, you can implement that.